src/14-curated-discovery/data.md

```diff
@@ -141,3 +151,36 @@ puts JSON.pretty_generate(partners.map do |p|
   }
 end)
 ```
+
+### Serialize marketing collections
+
+Again, we opt to use collector-facing terminology rather than Artsy internal
+jargon, on the assumption that this would perform better with the embedding
+models. We also want to be consistent in the field names we use for genes. For
+example:
+
+- category → **group**
+- genes → **categories**
+
+```ruby
+# Get all marketing collections with a description
+collections = MarketingCollection.where("description is not null and description <> ''").where(published: true)
```

Comment:
**suggestion:** We should consider changing this logic…

I was looking at the data dump and noticing all the html markup. This struck me as different from e.g. the artist blurb field that we extract into `artists.json` — there the text is markdown, which implies fewer tokens and is a bit more readable. Honestly I'm not sure how much difference that makes to the embedding model (other than token cost) but I was wondering if we had a similar markdown version of the `description`. 

And the answer is: yes, we also have a `description_markdown`. But the interesting thing (I think I knew this at one point but had to re-learn this just now) is that `description` is a legacy field from when these collections were [migrated](https://artsy.github.io/blog/2022/05/09/deprecating-kaws/) from a dedicated microservice called Kaws. Now that they live in Gravity, it's actually `description_markdown` which is [exposed to admins](https://api.artsy.net/admin/curated_marketing_collection/9e3af5d3-da56-4eaf-89b9-fcf865996e12).

Bottom line, `description_markdown` is the more up-to-date source of truth; covers 90% of collections instead of 80%; and uses fewer tokens to boot.